src/clr_bundle_add.c

```diff
@@ -53,6 +53,7 @@ static void print_help(const char *name)
 	printf("   -l, --list              List all available bundles for the current version of Clear Linux\n");
 	printf("   -x, --force             Attempt to proceed even if non-critical errors found\n");
 	printf("   -S, --statedir          Specify alternate swupd state directory\n");
+	printf("   -V, --verify-signatures Enable signature verification, default value is disabled\n");
```

Comment:
Probably not worth mentioning "default value is disabled", because enabling verification can be set with ./configure at build time.